organizations/backends.py

```diff
@@ -1,8 +1,24 @@
+import collections
+import logging
+
 from django.conf import settings
 from django.contrib.auth.backends import ModelBackend
 from django.core.exceptions import PermissionDenied
-from openedx.core.djangoapps.theming.helpers import get_current_site
-from openedx.core.djangoapps.site_configuration.helpers import is_site_configuration_enabled, get_value
+
+try:
+    from openedx.core.djangoapps.theming.helpers import get_current_site
+    from openedx.core.djangoapps.site_configuration.helpers import is_site_configuration_enabled, get_value
+except ImportError:
+    # we mock them out in test as we don't have edx-platform
+    from os import getenv
+    if 'test' in getenv('DJANGO_SETTINGS_MODULE'):
+        # set these so they are mockable from tests
+        def get_current_site(): pass
+        def get_value(_): pass
+        def is_site_configuration_enabled(): pass
```

Comment:
I agree with John here this should be mocked by tests instead of being in production code and potentially breaking the code silently.